packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -910,12 +1006,28 @@ class PerformanceOverlayLayer extends Layer {
   }
 }
 
+/// The signature of the callback added in [Layer.addCompositionCallback].
+typedef CompositionCallback = void Function(ContainerLayer);
```

Comment:
Related to the above, why is the argument not typed as ContainerLayer and not just Layer?